spec/commands/doctor_spec.rb

```diff
@@ -1,11 +1,25 @@
 # frozen_string_literal: true
 
+require "find"
 require "stringio"
 require "bundler/cli"
 require "bundler/cli/doctor"
 
 RSpec.describe "bundle doctor" do
   before(:each) do
+    install_gemfile! <<-G
+      source "file://#{gem_repo1}"
+      gem "rack"
+    G
+
+    stat = double("stat")
```

Comment:
Do we need these to be applied for every test in the file? i would wrap these and the check file permission specs in a `describe` so we keep everything close and only use these we're needed.

Reply:
We need that double on line 15 to be applied for every test, because we need the existing tests to be able to stat their files and see that they were created by the current user (Process.uid)